flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaTestBase.java

```diff
@@ -152,7 +152,7 @@ public static void startClusters(boolean secureMode) throws Exception {
 
     public static void startClusters(KafkaTestEnvironment.Config environmentConfig)
             throws Exception {
-        kafkaServer = constructKafkaTestEnvionment();
+        kafkaServer = constructKafkaTestEnvironment();
```

Comment:
Nit: Can you make the renaming on a separate commit as hotfix?

Reply:
I already rebased my commits and made this change in a separate commit as other PRs, do you mean I should submit another PR for this hotfix?